datadog-dashboards-dashboard-handler/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.0.0 / 2021-02-16
+
+* [Fixed] Bump common-python dependencies in all resources. See [#117](https://github.com/DataDog/datadog-cloudformation-resources/pull/117).
+* [Fixed] Bump datadog-api-client version. See [#114](https://github.com/DataDog/datadog-cloudformation-resources/pull/114).
+* [Fixed] Bump cloudformation-cli-python-plugin version. See [#111](https://github.com/DataDog/datadog-cloudformation-resources/pull/111).
+* [Fixed] Fix the user agent header when using the python common package. See [#109](https://github.com/DataDog/datadog-cloudformation-resources/pull/109).
```

Comment:
I'm a bit torn if we should include these in the Changelog. Since (AFAIU) these are changes to the common package, not the resource? Wondering if we should just leave the first line and state the version we bumped to? WDYT?

Reply:
You have a fair point and I agree. Including the top PR alone should suffice.